src/components/VitalsSideBar.vue

```diff
@@ -1,12 +1,12 @@
 <script>
 import VitalStat from './VitalStat.vue';
 /**
- * Displays vital stats such as health, willpower etc.
+ * Displays dependent stats in bar located to the left side, also contains algorithm
+ * how values are computed
```

Comment:
hmm to je pravda... co je podla mna zle... toto by malo byt povedane v datach ako sa co ma pocitat a nie zasite v komponente